terraform/subscriptions/s940/c2/pre-clusters/redis.tf

```
@@ -1,6 +1,6 @@
 module "rediscache" {
   source              = "../../../modules/redis_cache"
-  for_each            = { for k in jsondecode(nonsensitive(data.azurerm_key_vault_secret.this.value)).clusters : k.name => k }
+  for_each            = { for k, v in var.aksclusters : k => v }
```

```suggestion
  for_each            = var.aksclusters
```
Denne gjør vel ingenting?